src/eu/packages/eu-component-icon/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl-twig/eu-component-icon",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EU Icon",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@ecl/ec-resources-icons": "^2.0.0"
```

Comment:
@nimek2 , the thing here is that you are providing a component as part of an ecosystem, in this case is EU so your resources should come from the EU implementation of ECL in this case, and all of your imports then would use the resources provided by this package, not the EC one.. ;)

For the rest yes, the problem about the icons not rendering is due to webpack conf, i will fix that, once you fix this EC-EU confusion the pull request will be merged, we already reviewed the EC version.
Thanks!

Reply:
I changed and commited the resource from ec-resources-icons to eu-resources-icons, is it enough?